app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -981,7 +981,13 @@ class BrowserTabFragment :
 
     override fun onResume() {
         super.onResume()
+
+        if (omnibar.omnibarPosition != settingsDataStore.omnibarPosition) {
+            requireActivity().recreate()
```

Comment:
The activity needs to be restarted because the fragment needs to reinitialize the views from the other omnibar (top/bottom).